code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -158,6 +158,8 @@
 			health = 100 - getOxyLoss() - getToxLoss() - getFireLoss() - getBruteLoss()
 		diag_hud_set_status()
 		diag_hud_set_health()
+		ear_damage = null

```

Comment:
AI code. Why am I suprised?
Still, though, should be 0, it's _supposed_ to be a number..